testcases/crypto/des3.h

```diff
@@ -862,50 +856,50 @@ static struct des3_test_vector des3_ofb64_tv[] = {
 #endif
```

Comment:
According to the new coding style the "# define NUM_OF_PUBLISHED_TESTSUITES" should probably be without a blank.
I should have made it the correct was with my patch already, sorry....